sources/packages/web/src/main.ts

```diff
@@ -25,6 +25,7 @@ import InputNumber from "primevue/inputnumber";
 import ProgressSpinner from "primevue/progressspinner";
 import { AppConfigService } from "./services/AppConfigService";
 import configValidationRules from "./validators/ValidatorConfigUI";
+import "./assets/css/institution.css";
```

Comment:
Just as a recommendation, is it a global style, or is it only needed for the locations page?
If everything in the style belongs only to the scope of the location page, please considering moving it to a scoped CSS block inside locations view.